libs/ballot-interpreter/src/hmpb-rust/timing_marks.rs

```diff
@@ -1691,20 +1693,24 @@ pub fn rect_could_be_timing_mark(geometry: &Geometry, rect: &Rect) -> bool {
     let min_timing_mark_width_multiplier = if is_near_left_or_right_edge {
         0.20
     } else {
-        0.75
+        0.5
     };
     let min_timing_mark_height_multiplier = if is_near_top_or_bottom_edge {
         0.20
     } else {
-        0.75
+        0.5
     };
 
+    // rotation can cause the height to be slightly larger than expected
+    const MAX_EXTRA_HEIGHT_DUE_TO_ROTATION: u32 = 3;
+
     let min_timing_mark_width =
         (timing_mark_size.width * min_timing_mark_width_multiplier).floor() as u32;
     let min_timing_mark_height =
         (timing_mark_size.height * min_timing_mark_height_multiplier).floor() as u32;
     let max_timing_mark_width = (timing_mark_size.width * 1.50).round() as u32;
-    let max_timing_mark_height = (timing_mark_size.height * 1.50).round() as u32;
+    let max_timing_mark_height =
+        (timing_mark_size.height * 1.50).round() as u32 + MAX_EXTRA_HEIGHT_DUE_TO_ROTATION;
```

Comment:
It seems a little odd that we're already multiplying by 1.5 and then still need to add 3. Wouldn't the rotated ones that you showed me easily fall within the 1.5x height bound? If not, should we just increase 1.5 instead of adding 3?

Reply:
I'm okay with updating the 1.5x scale instead 👍